container_pipeline/workers/build.py

```diff
@@ -87,7 +87,9 @@ def handle_build_success(self, job):
 
     def handle_build_failure(self, job):
         """Handle build failure for job"""
-        self.queue.put(json.dumps(job))
+        job.pop('action', None)
+        job['action'] = "build_failure"
```

Comment:
But its okay, if an extra line of operation is clearly indicating the goal of operation.